_docs_integrate/data-model-overview.md

```diff
@@ -323,26 +323,46 @@ Thus, it is not possible for an Identity to have an unshared RelationshipAttribu
+| requestReference      | `string` \| `undefined` | The `id` of the [LocalRequest](#localrequest) the LocalAttribute was received in/sent with. If this is set, `notificationReference` must be undefined.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| notificationReference | `string` \| `undefined` | The `id` of the [LocalNotification](#localnotification) the LocalAttribute was received in/sent with. If this is set, `requestReference` must be undefined.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| sourceAttribute       | `string` \| `undefined` | If the [LocalAttribute](#localattribute) is a copy of a RepositoryAttribute, then this property contains the `id` of the RepositoryAttribute. Also, if a RelationshipAttribute is shared with a third party, i.e. in the context of a Relationship that is different to the one the RelationshipAttribute was initially created in, this ThirdPartyRelationshipAttribute will link to the source RelationshipAttribute. Note that `sourceAttribute` is only defined for the peer who also has the Attribute this property links to, i.e. for IdentityAttribute the owner and for ThirdPartyRelationshipAttributes the peer who is in the Relationship the source Attribute is copied from. |
+
+### LocalAttributeDeletionInfo
+
+The LocalAttributeDeletionInfo holds information about the deletion status of the counterpart of a shared Attribute, as well as the point in time of that deletion.
+If the Attribute's owner requested the deletion of an shared Attribute via [DeleteAttributeRequestItem](#deleteattributerequestitem) from the peer and they accepted, the peer shared Attribute of the peer will be set to the `deletionStatus` `"ToBeDeleted"` with the `deletionDate` the peer sent in their Response.
+Accordingly, the own shared Attribute of the owner will move to the `deletionStatus` `"ToBeDeletedByPeer"` with the same `deletionDate`.
```

Comment:
What would happen to the deletionStatus and deletionDates if the peer had already deleted the Attribute after receiving the Request but before accepting it?